docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,847 @@
+{
```

Comment:
<p>tqdm isn't really doing anything here and the reader might think that they need to install it. Please remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/872/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>